tools/landmark_registration/landmark_registration.xml

```diff
@@ -1,41 +1,50 @@
-<tool id="ip_landmark_registration" name="Landmark Registration" version="0.0.2">
-    <description>Landmark Registration</description>
+<tool id="ip_landmark_registration" name="Landmark Registration" version="0.0.3" profile="20.05">
+    <description>estimates the affine transformation matrix</description>
     <requirements>
-        <requirement type="package" version="0.14.2">scikit-image</requirement>
-        <requirement type="package" version="0.23.4">pandas</requirement>
-        <requirement type="package" version="1.15.4">numpy</requirement>
-        <requirement type="package" version="0.15.1">tifffile</requirement>
+        <requirement type="package" version="0.18.1">scikit-image</requirement>
```

Comment:
Btw. we have a bot that bumps those dependencies, we could enable this bot for this repo as well. Just let me know.

Reply:
I do not know these concepts. Sorry, but could you give some more hints?

Reply:
This PR for example: https://github.com/galaxyproject/tools-iuc/pull/4405